### PR TITLE
Remove comments in `nuget.yml`

### PR DESCRIPTION
**Pull request by cschuchardt88**
# Description
Fixed typo in `nuget.yml`, Comments create an error in `msbuild`. Also changed the path.

## Type of change

- [ ] Optimization (the change is only an optimization)
- [ ] Style (the change is only a code style for better maintenance or standard purpose)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Jim8y — DISMISSED** [review]

File: .github/workflows/nuget.yml [hunk 1]
```diff
@@ -34,13 +34,13 @@ jobs:
       run: |
         dotnet pack ./neo.sln \
         --configuration Release \
-        --output ./sbin \
-        --verbosity normal \                        # Normal verbosity level
-        -p:VersionPrefix=${{ env.APP_VERSION }}     # Set the version prefix from tag_name
+        --output ./bin/packages \
```
Comment (vncoelho) [inline .github/workflows/nuget.yml:37]: Does the path need to be modified?

Comment (cschuchardt88) [reply .github/workflows/nuget.yml:37]: More consistent with the rest of the `yml` files

Comment (vncoelho) [reply .github/workflows/nuget.yml:37]: Which yml files?
I did not understand, the paths are different, right?

Comment (cschuchardt88) [reply .github/workflows/nuget.yml:37]: > Which yml files?

All the ones I been making. `release.yml`, `main.yml`, `docker.yml`, Also `C#` solution

> I did not understand, the paths are different, right?

They all fall under `bin` that will be the place you they will be stored under for all builds. So it isn't some random location a developer picked.

Comment (shargon) [reply .github/workflows/nuget.yml:37]: I'm with @vncoelho this is necessary?

Comment (cschuchardt88) [reply .github/workflows/nuget.yml:37]: Yes the comments, On the command-line will break the workflow and have the step `fail`.

**Review by shargon — COMMENTED** [review]
As I see, nothing to fix here, it works already

**Comment by cschuchardt88** [comment]
The `comments` is the problem

**Review by shargon — APPROVED** [review]

**Review by Jim8y — APPROVED** [review]

**Review by vncoelho — APPROVED** [review]

**Pull request merged**